config/locales/en.yml

```diff
@@ -629,6 +631,13 @@ en:
       passwords_dont_match: "The two passwords entered don't match."
       success: The user was updated.
       error: A problem kept the user from being updated.
+      photo:
+        reason: Your photo will be used for your "proof of membership" credentials.
```

Comment:
general thoughts to help make the purpose of the photo clearer:
- add more information about how this photo can/will be used: 
  - you can show your SHF proof of membership to the public on your own website and social media   (maybe link to other places on the total SHF site where this is mentioned?)
  - you can print this out as a membership card
  - the public will be able to see this picture of you on the website

We can talk about this during the client review and see if these ideas are actually appropriate/needed

Reply:
These descriptions are just placeholders, and I want @thesuss to put the "real" descriptions together.  Links to an appropriate page(s) on the standard site would be good, of course, in describing what "proof of membership" is and how it is to be used.

(also, the public having visibility to the member photo comes later, of course).